readme.md

```diff
@@ -61,6 +61,50 @@ Type: `number`
 
 Reduce the number of distinct colors in each output GIF to num or less. Num must be between 2 and 256.
 
+##### lossy
+
+Type: `number`\
+Default: `20`
+
+Alter image colors to shrink output file size at the cost of artifacts and noise. Lossiness determines how many artifacts are allowed; higher values can result in smaller file sizes, but cause more artifacts.
+
+##### colorMethod
+
+Type: `string`\
+Default: `diversity`
+
+Determine how a smaller colormap is chosen. `diversity`, the default, is xv(1)’s diversity algorithm, which uses a strict subset of the existing colors and generally produces good results. `blend-diversity` is a modification of this: some color values are blended from groups of existing colors. `median-cut` is the median cut algorithm described by Heckbert.
```

Comment:
Can you linkify some of the terms here so the user can quickly read more about the various methods?